app/controllers/schools_controller.rb

```diff
@@ -1,10 +1,23 @@
 # frozen_string_literal: true
 
 class SchoolsController < ApplicationController
+  layout 'info_site/application'
   
   def index
-    @schools = School.active
-
+    # Group by first letter of the school.
+    @schools = School.active.group_by { |i| i.name[0] }
+    @schools_arr = []
+    @schools.each do |key, value|
```

Comment:
Would be great if you used descriptive names instead of `key` & `value` — I don't really know what they are.